docs/content/direct/testing.md

```diff
@@ -15,29 +15,26 @@ make test
 There are currently three integration tests. Contributors can run them. There is also a GitHub Actions workflow (in `.github/workflows/pr-test-integration.yml`) that runs these tests.
 
 These tests require you to already have `etcd` on your `$PATH`.
-See https://github.com/kubernetes/kubernetes/blob/v1.29.10/hack/install-etcd.sh for an example of how to do that.
+See [https://github.com/kubernetes/kubernetes/blob/v1.29.10/hack/install-etcd.sh](https://github.com/kubernetes/kubernetes/blob/v1.29.10/hack/install-etcd.sh) for an example of how to do that.
 
 To run the tests sequentially, issue a command like the following.
 
-```shell
+```
 CONTROLLER_TEST_NUM_OBJECTS=24 go test -v ./test/integration/controller-manager &> /tmp/test.log
 ```
 
-If `CONTROLLER_TEST_NUM_OBJECTS` is not set then the number of objects
-will be 18. This parameterization by an environment variable is only a
-point-in-time hack, it is expected to go away once we have a test that
-runs reliably on a large number of objects.
+If `CONTROLLER_TEST_NUM_OBJECTS` is not set then the number of objects will be 18. This parameterization by an environment variable is only a point-in-time hack, it is expected to go away once we have a test that runs reliably on a large number of objects.

```

Comment:
While it is valid to put a whole paragraph in a single long source line, this makes review harder. A review comment is attached to a source line, so when that is a whole paragraph it is harder to see what part of the paragraph is being commented upon.